tests/integration/logging.tcl

```diff
@@ -21,15 +21,20 @@ if {$system_name eq {linux} || $system_name eq {darwin}} {
             }
         }
     }
-}
 
-set server_path [tmpdir server1.log]
-start_server [list overrides [list dir $server_path]] {
-    test "Crash report generated on SIGABRT" {
-        set pid [s process_id]
-        exec kill -SIGABRT $pid
-        set pattern "*STACK TRACE*"
-        set result [exec tail -1000 < [srv 0 stdout]]
-        assert {[string match $pattern $result]}
+    if {!$::valgrind} {
```

Comment:
we may want to add a comment on why it is skipped (or maybe rely on the details from the blame log)

Reply:
Will add a message